example/fgGraffTypeVis.jl

```diff
@@ -0,0 +1,126 @@
+# Tutorial on conventional 2D SLAM example
+# This tutorial shows how to use some of the commonly used factor types
+# This tutorial follows from the ContinuousScalar example from IncrementalInference
+using GraffSDK
+using GraffSDK.DataHelpers
+using ProgressMeter
+using UUIDs
+using IncrementalInference
+using RoME
+using Arena.Amphitheatre
+using Colors
+##
+
+# 1a. Create a Configuration
+config = loadGraffConfig();
+#Create a hexagonal sessions
+config.sessionId = "HexVisDemo"*string(uuid4())[1:6]
+
+@info "Session backlog (queue length) = $(getSessionBacklog())"
+
+# 2. Confirm that the robot already exists, create if it doesn't.
+println(" - Creating or retrieving robot '$(config.robotId)'...")
+robot = nothing
+if isRobotExisting()
+    println(" -- Robot '$(config.robotId)' already exists, retrieving it...")
+    robot = getRobot();
+else
+    # Create a new one programatically - can also do this via the UI.
+    println(" -- Robot '$(config.robotId)' doesn't exist, creating it...")
+    newRobot = RobotRequest(config.robotId, "My New Bot", "Description of my neat robot", "Active");
+    robot = addRobot(newRobot);
+end
+println(robot)
+
+# 3. Create or retrieve the session.
+# Get sessions, if it already exists, add to it.
+println(" - Creating or retrieving data session '$(config.sessionId)' for robot...")
+session = nothing
+if isSessionExisting()
+    println(" -- Session '$(config.sessionId)' already exists for robot '$(config.robotId)', retrieving it...")
+    session = getSession()
+else
+    # Create a new one
+    println(" -- Session '$(config.sessionId)' doesn't exist for robot '$(config.robotId)', creating it...")
+    newSessionRequest = SessionDetailsRequest(config.sessionId, "A test dataset demonstrating data ingestion for a wheeled vehicle driving in a hexagon.", "Pose2")
+    session = addSession(newSessionRequest)
+end
+println(session)
+
+# 4. Drive around in a hexagon
+println(" - Adding hexagonal driving pattern to session...")
+@showprogress for i in 1:6
+    deltaMeasurement = [1.0;0;pi/3]
+    pOdo = Float64[0.1 0 0; 0 0.1 0; 0 0 0.1]
+    println(" - Measurement $i: Adding new odometry measurement '$deltaMeasurement'...")
+    @time addOdometryMeasurement(deltaMeasurement, pOdo)
+end
+
+# 5. Now lets add a couple landmarks
+# Ref: https://github.com/dehann/RoME.jl/blob/master/examples/Slam2dExample.jl#L35
+response = addVariable("l1", "Point2", ["LANDMARK"])
```

Comment:
at some point we should upgrade this to the GraffSDK.addVariable(...) and GraffSDK.addFactor(..) API

Reply:
on addFactor for example (the variables are fine -- sorry)
https://github.com/JuliaRobotics/Caesar.jl/blob/4505d881ba73c4c88dc6b6007fc308d2a5beae0e/examples/web/01_duplicate_hexagonal.jl#L82